### PR TITLE
Adds productId

### PR DESCRIPTION
**Pull request by lucasayb**
#### What is the purpose of this pull request?
Adds productId for `orderForm` items

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Adds productId for item listing
+### Added
```
Comment (klzns) [inline CHANGELOG.md:11]:
```suggestion
```

**Comment by klzns** [comment]
@lucasayb can you provide a link workspace?

**Comment by lucasayb** [comment]
> @lucasayb can you provide a link workspace?

Yes, here it's https://lucas--codeby.myvtex.com/

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Adds productId for item listing
```
Comment (thalytafabrine) [inline CHANGELOG.md:10]:
```suggestion
- `productId` for item listing.
```

**Pull request merged**